### ISSUE TITLE
Update to Elm v0.19

### ISSUE THREAD

**Issue by prikhi** [elm-community/builtwithelm#179]
It would be nice to be able to use this repo as an example of a v0.19 app.

**Comment by joelclermont** [comment]
Agreed, I was thinking of hacking at this at the next Elm meetup in February.
